xarray/core/accessor_str.py

```diff
@@ -803,7 +951,21 @@ def islower(self) -> T_DataArray:
         Returns
         -------
         islower : array of bool
-            Array of boolean values with the same shape as the original array.
+            Array of boolean values with the same shape as the original array indicating whether each
+            element of the string array is lowercase (True) or not (False).
```

Comment:
I think using "all" somehow would make the purpose a bit easier to understand (though the examples of course help):
```suggestion
            Array of boolean values with the same shape as the original array indicating whether
            all characters of each element of the string array are lowercase (True) or not (False).
```